code/game/objects/structures/watercloset.dm

```diff
@@ -141,6 +197,27 @@
 
 
 /obj/structure/urinal/attackby(obj/item/I, mob/user, params)
+	if(iswrench(I))
+		if(anchored)
+			user.visible_message("[user] begins disconnecting [src]...", "<span class='notice'>You begin to disconnect [src]...</span>")
```

Comment:
Not arguing to keep it as such, but do want to point out a LOT of things in the code are missing spans for these kinds of messages. Even the message when you completed removing the lid from the toilet cistern prior to this PR was like this, as is also the case for things like pipes.

I'll still fix these messages, but at some point we'll probably want to go through and fix the other issues to avoid accidental referencing to improper styling.